test/tasks/processIssueComment/probot-app-e2e.test.js

```diff
@@ -22,6 +22,19 @@ describe('All Contributors app - End to end', () => {
         return true
     }
 
+    const verifyBodyForPull = body => {
```

Comment:
See comment about, if we're able to move the logic into the OptionsConfig class we can just have an test on that class (no need for more e2e ones) thoughts?